site/content/docs/5.3/components/card.md

```diff
@@ -300,6 +324,12 @@ You can quickly change the text alignment of any card—in its entirety or speci
 
 Add some navigation to a card's header (or block) with Boosted's [nav components]({{< docsref "/components/navs-tabs" >}}).
 
+{{< design-callout-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications.
```

Comment:
So I guess the second variant is fine ? 😅

Reply:
No, it's not. I'll put the message in plural.